website/contributing/release-troubleshooting.md

```diff
@@ -42,3 +42,50 @@ git pull <remote> <branch> --tags
 # Make sure you push any local tags you added/removed
 git push <remote> <branch> --follow-tags
 ```
+
+### The Tag Has Been Pushed but NPM Failed to Publish React Native
+
+The release process takes various steps, but they can be summarised with these two:
+
+1. Create a version tag on the GitHub repository
+2. Publish all the artifacts on Maven and NPM
+
+Step 2 takes some time because it has to build everything we need to publish and it can happen that some builds fail or there are other issues.
+
+In case Step 1 completes successfully but step 2 fails, the repo is in a dirty situation and you need to address that.
```

Comment:
```suggestion
In case Step 1 completes successfully but Step 2 fails, the code in the stable branch is set to the new version, but it has failed to rollout. This means that we need to do some extra steps to be able to retry to publish this version.
```